Hirsch_sites/EGRET/EGRETtoANA.R

```diff
@@ -0,0 +1,90 @@
+EGRETtoANA <- function(egret.files, ANA.dir) {
+  
+  library(dplyr)
+  library(EGRET)
+  library(loadflex)
+  
+  # read the RData files and confirm that they're EGRET eLists
+  elists <- lapply(egret.files, function(egret.file) {
+    elist.name <- load(egret.file)
+    egret.elist <- get(elist.name)
+    if(!all.equal(names(egret.elist), c('INFO','Daily','Sample','surfaces')) || class(egret.elist) != 'egret') {
+      stop("are you sure this file contains an eList?")
```

Comment:
since this happens for each file, should the error message include the file name that does not appear to be an eList?